adafruit_as726x.py

```diff
@@ -550,7 +559,7 @@ def _uart_xfer(self, cmd):
             return resp.rstrip(b" OK\n")
         return None
 
-    def _virtual_read(self, addr):
+    def _virtual_read(self, addr: int) -> int:
```

Comment:
It looks like the last `elif` say that it casts to a `float` so the return should be `float` which will also account for the other `int` return types

Reply:
Yup. Adjusted here as well.